### PR TITLE
Add styling to homepage and include package repos

### LINKED ISSUE #31: Styling deno.land/x/ and clickable links to repos
Page https://deno.land/x/ is useful but lacks some minimal styling and clickable links to the github repositories. I would be happy to add that.

### PR DESCRIPTION
**Pull request by rista404**
@ry I tried to keep the code changes as minimal as possible (not using any external packages/tools), but if you want I can make a followup PR with some template engine or something.

I also tried to run this locally using aws's `sam local start-api` but with no luck, getting an error. So ended up writing `open_homepage.js` to test the website. 😅 I can remove if it's not actually needed. 

The live html is on https://deno-packages.ristic.tech, I followed deno.land's style — let me know if you want any changes done. 💅 

Closes #31

### CODE REVIEW

File: src/database.json [hunk 1]
```diff
@@ -1,27 +1,98 @@
...
-  "watch": "https://raw.githubusercontent.com/jinjor/deno-watch/${b}/",
-  "task_runner": "https://raw.githubusercontent.com/jinjor/deno-task-runner/${b}/",
-  "ws": "https://raw.githubusercontent.com/denoland/deno_std/${b}/ws/",
-  "io": "https://raw.githubusercontent.com/denoland/deno_std/${b}/io/",
-  "media_types": "https://raw.githubusercontent.com/denoland/deno_std/${b}/media_types/",
-  "textproto": "https://raw.githubusercontent.com/denoland/deno_std/${b}/textproto/",
-  "postgres": "https://raw.githubusercontent.com/bartlomieju/deno-postgres/${b}/"
+  "net": {
+    "url": "https://raw.githubusercontent.com/denoland/deno_std/${b}/net/",
+    "repo": "https://github.com/denoland/deno_std/tree/master/net"
```
Comment (rista404) [inline src/database.json]: Also, I couldn't find this package in `deno_std`

File: src/database.json [hunk 2]
```diff
@@ -1,27 +1,98 @@
...
+  },
+  "http": {
+    "url": "https://raw.githubusercontent.com/denoland/deno_std/${b}/http/",
+    "repo": "https://github.com/denoland/deno_std/tree/master/http"
+  },
```
Comment (j-f1) [inline src/database.json]: Hm, could this be reduced to a simple `{ "github": "denoland/deno_std/http" }`?

```ts
const [user, repo, ...components] = mod.github.split('/')
const path = components.length ? components.join('/') + '/' : ''
return {
  url: `https://raw.githubusercontent.com/${user}/${repo}/${branch}/${path}`,
  repo: path ? `https://github.com/${user}/${repo}/tree/master/${path}` : `https://github.com/${user}/${repo}/`
}
```

File: src/homepage.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+		<p>This is a URL redirection service for Deno scripts.</p>
+		<p>
+			The basic format is <code>https://deno.land/x/PACKAGE_NAME@BRANCH/SCRIPT.ts</code>. If you leave out the branch, it will default to master.
+		</p>
+
+		<h2 id="available-packages">Available Packages</h2>
+
+		<ul class="packages">
+			${
+				Object.entries(DATABASE)
```
Comment (chrisdothtml) [inline src/homepage.js]: Maybe sort the list here so they show up alphabetized?

File: src/database.json (hunk 2 above)
Comment (rista404) [reply src/database.json]: That way we can't have links to `gitlab.com` or other websites. 🤔 
For example, `database.json` previously had a link to `gist.github.com` script.

Comment (j-f1) [reply src/database.json]: You could still have a fallback to `url` and `repo` if the package isn’t on GitHub.

If another hosting platform becomes popular, then a special case could be added for it.

Comment (j-f1) [reply src/database.json]: Also: `{ "std": "http" }`

```ts
return {
  url: `https://raw.githubusercontent.com/denoland/deno_std/${branch}/${mod.std}`,
  repo: `https://github.com/denoland/deno_std/tree/master/${mod.std}`
}
```

File: src/homepage.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
+const DATABASE = require("./database.json");
+
+const LOGO_PATH =
+  "https://raw.githubusercontent.com/denoland/deno/master/website/deno_logo.png";
+
+const homepageHTML = `
```
Comment (j-f1) [inline src/homepage.js]: If you mark this as HTML, Prettier will format it as such:


```suggestion
const homepageHTML = /* HTML */ `
```

Comment (j-f1) [reply src/homepage.js]: Will open a PR.

**Pull request merged**